apps/ui/src/networks/common/graphqlApi/index.ts

```diff
@@ -57,10 +57,18 @@ function getProposalState(
   proposal: ApiProposal,
   current: number
 ): ProposalState {
+  // we have broken types, we should unify, this is quick fix for Nimbora PR
+  // those values are actually strings
+  // https://github.com/snapshot-labs/sx-monorepo/pull/529/files#r1691071502
+  const quorum = BigInt(proposal.quorum);
+  const scoresTotal = BigInt(proposal.scores_total);
+  const scoresFor = BigInt(proposal.scores_1);
+  const scoresAgainst = BigInt(proposal.scores_2);
```

Comment:
These may fail for offchain proposals if `score_total` has decimals 🙈  same issue as https://github.com/snapshot-labs/sx-monorepo/pull/529

I think better to use `toBigIntOrNumber` instead of `BigInt` for now

Reply:
This is inside `common/graphqlApi` that is only used for onchain spaces and they all use `string` without decimals.

Reply:
ahh got it 😄